tests/marine/test_marine.py

```diff
@@ -5,44 +5,53 @@
 from typing import List, Union, Optional, Dict
```

Comment:
I think some more tests are required:
1. Test that different encapsulations affect BPF validation
2. Test that different encapsulations affect parsing
3. Show that display filter validation works for types of different encapsulation (for example, add a test that shows both `radiotap` and `ethernet` are valid display filters without needing to specify encapsulation)

Reply:
I added a test that checks for effect on BPF validation and tests that affect parsing.
I could not add a test that checks the 3rd requirement as the display filter validation is not affected by encapsulation type, as the display filter is not dependent on it during compilation.